keyboards/mxss/mxss.h

```diff
@@ -0,0 +1,206 @@
+	{ k30, KC_NO, k32, k33, k34, k35, k36, k37, k38, k39, k3A, k3B, k3C, k3D, k3E }, \
+	{ k40, k41, k42, k43, KC_NO, KC_NO, k46, KC_NO, k48, KC_NO, k4A, k4B, k4C, k4D, k4E }, \
+}
+//ANSI, splitBS, 6.25u
+#define LAYOUT_splitBS( \
+    	k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0A, k0B, k0C, k0D, k0E, \
+    	k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1A, k1B, k1C, k1D, k1E, \
+	k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2A, k2B,   k2D,    k2E, \
+	k30,    k32, k33, k34, k35, k36, k37, k38, k39, k3A, k3B, k3C, k3D, k3E, \
+	k40, k41, k42,                k46,                k4A, k4B, k4C, k4D, k4E \
+) \
+{ \
+    	{ k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0A, k0B, k0C, k0D, k0E }, \
+    	{ k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1A, k1B, k1C, k1D, k1E }, \
+	{ k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2A, k2B, KC_NO, k2D, k2E }, \
+	{ k30, KC_NO, k32, k33, k34, k35, k36, k37, k38, k39, k3A, k3B, k3C, k3D, k3E }, \
+	{ k40, k41, k42, KC_NO, KC_NO, KC_NO, k46, KC_NO, KC_NO, KC_NO, k4A, k4B, k4C, k4D, k4E }, \
+}
+//ANSI, splitBS, 7u
+#define LAYOUT_splitBS_7u( \
+    	k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0A, k0B, k0C, k0D, k0E, \
+    	k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1A, k1B, k1C, k1D, k1E, \
+	k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2A, k2B,   k2D,    k2E, \
+	k30,    k32, k33, k34, k35, k36, k37, k38, k39, k3A, k3B, k3C, k3D, k3E, \
+	k40, k41, k42,                k46,                 k4B,  k4C, k4D, k4E \
+) \
+{ \
+    	{ k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0A, k0B, k0C, k0D, k0E }, \
+    	{ k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1A, k1B, k1C, k1D, k1E }, \
+	{ k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2A, k2B, KC_NO, k2D, k2E }, \
+	{ k30, KC_NO, k32, k33, k34, k35, k36, k37, k38, k39, k3A, k3B, k3C, k3D, k3E }, \
+	{ k40, k41, k42, KC_NO, KC_NO, KC_NO, k46, KC_NO, KC_NO, KC_NO, KC_NO, k4B, k4C, k4D, k4E }, \
+}
+//ANSI, normalBS, split space
+#define LAYOUT_splitBS_splitSpace( \
+    	k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0A, k0B, k0C, k0D, k0E, \
+    	k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1A, k1B, k1C, k1D, k1E, \
+	k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2A, k2B,    k2D,   k2E, \
+	k30,     k32, k33, k34, k35, k36, k37, k38, k39, k3A, k3B, k3C, k3D, k3E, \
+	k40, k41, k42,      k43,      k46,      k48,      k4A, k4B, k4C, k4D, k4E \
+) \
+{ \
+    	{ k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0A, k0B, k0C, k0D, k0E }, \
+    	{ k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1A, k1B, k1C, k1D, k1E }, \
+	{ k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2A, k2B, KC_NO, k2D, k2E }, \
+	{ k30, KC_NO, k32, k33, k34, k35, k36, k37, k38, k39, k3A, k3B, k3C, k3D, k3E }, \
+	{ k40, k41, k42, k43, KC_NO, KC_NO, k46, KC_NO, k48, KC_NO, k4A, k4B, k4C, k4D, k4E }, \
+}
+//ISO, normalBS, 6.25u
+#define LAYOUT_ISO( \
```

Comment:
Is this the same layout as https://github.com/qmk/qmk_firmware/blob/master/layouts/default/60_iso/default_60_iso/keymap.c

if it is, can you rename it to LAYOUT_60_iso?

Reply:
@mechmerlin The keyboard isn't a 60%. It's a 65%, but not the most-common 65% layout.

https://geekhack.org/index.php?topic=94986.0